draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1028,12 +1028,14 @@ a cryptographic hash function that outputs b bits.
 ## Security considerations {#hash2base-sec}
 
 For security, hash2base should be collision resistant, and should map its
-input to a uniformly random element of F.
-
-To ensure sufficient collision resistance, the number of bits output by
-the hash function H should be b >= 2 * k, where k is the target security
-level in bits. For example, for 128-bit security, b >= 256 bits; in this
-case, SHA256 would be an appropriate choice for H.
+input to a uniformly random element of F. To this end, hash2base requires 
+a cryptographic hash function H which satisfies the following properties:
+
+1. The number of bits output by H should be b >= 2 * k for sufficient collision 
+resistance, where k is the target security level in bits. (This is needed for a 
+birthday bound of approximately 2^(-k).) For example, for 128-bit security, b >= 256 
+bits; in this case, SHA256 would be an appropriate choice for H.
+2. H must output a uniformly random bit string. That is, it must be a random oracle.
```

Comment:
It might be slightly more accurate to say

> H is modeled as a random oracle, so its output must be indistinguishable from a uniformly random bit string.

because of course SHA doesn't *really* output a uniformly random bit string... :)

Also, maybe it would be good to pull the "For example" sentence below point (2), to make clear that SHA-256 satisfies both requirements.

Reply:
Agreed -- fixed!